metricflow/query/resolver_inputs/query_resolver_inputs.py

```diff
@@ -117,31 +119,65 @@ def ui_description(self) -> str:
 
 
 @dataclass(frozen=True)
-class ResolverInputForWhereFilterIntersection(MetricFlowQueryResolverInput):
-    """An input that describes the where filter."""
+class ResolverInputForQueryLevelWhereFilterIntersection(MetricFlowQueryResolverInput):
+    """An input that describes the where filter for the query."""
 
     where_filter_intersection: WhereFilterIntersection
 
     @property
     @override
     def ui_description(self) -> str:
-        # TODO: Improve description.
         return (
-            "WhereFilter("
-            + mf_pformat(
-                [where_filter.where_sql_template for where_filter in self.where_filter_intersection.where_filters]
+            "WhereFilter(\n"
+            + indent_log_line(
+                mf_pformat(
+                    [where_filter.where_sql_template for where_filter in self.where_filter_intersection.where_filters]
+                )
             )
-            + ")"
+            + "\n)"
         )
 
 
+@dataclass(frozen=True)
+class VirtualResolverInputForWhereFilterIntersection(MetricFlowQueryResolverInput):
```

Comment:
Why "Virtual"? Confused what that's supposed to mean in this context 🤔

Reply:
NVM, thought about it more and removed that prefix.